### PR TITLE
Update easi

### PR DESCRIPTION
**Pull request by krenzland**
Can only be merged after: https://github.com/SeisSol/easi/pull/18

I will write documentation and fix the CI soon (and then convert to a standard PR).

### CODE REVIEW

**Comment by krenzland** [comment]
For a complete documentation, https://github.com/SeisSol/easi/pull/19 needs to be merged first.

File: Documentation/compilation.rst [hunk 1]
```diff
@@ -143,6 +143,10 @@ See section :ref:`Installing ASAGI <installing_ASAGI>`.
 
 .. _compiling-seissol:
 
+Installing easi
+---------------------------
+Follow the `installation instructions <https://easyinit.readthedocs.io/en/latest/index.html>`.
```
Comment (Thomas-Ulrich) [inline Documentation/compilation.rst:148]: There is no installation instruction there.
It also says:
>easi is a header-only library written in C++11. Hence, one simply needs (Only unit tests must be compiled with CMake.)
Maybe you want to copy-paste some content from here:
https://gitlab.lrz.de/thomas.ulrich/TuSeisSolScripts/-/blob/master/EasiRelated/README.md

**Review by Thomas-Ulrich — APPROVED** [review]
Nice, but I think we need to better document the installation of easi in the readthedocs of easi.

File: Documentation/compilation.rst (hunk 1 above)
Comment (krenzland) [reply Documentation/compilation.rst:148]: Hm, the corresponding PR is already merged but it doesn't seem to be building the documentation.

Comment (Thomas-Ulrich) [reply Documentation/compilation.rst:148]: ok, I see.
But don't you think that all users can figure out things like:
```
git clone git@github.com:uphoffc/ImpalaJIT
cd ImpalaJIT 
mkdir build && cd build
CC=mpicc CXX=mpiCC FC=mpif90 cmake ..  -DCMAKE_INSTALL_PREFIX=$HOME
make -j 4
make install
cd ../..
```
(I'm not sure, and detailing may prevent having to give further explanation).

Comment (krenzland) [reply Documentation/compilation.rst:148]: It is there, but you need to delete your cache - ctrl+F5 for most browsers.

File: CMakeLists.txt [hunk 2]
```diff
@@ -269,7 +230,7 @@ endif()
 find_package(PkgConfig REQUIRED) 
 if (ASAGI)
   # todo warn if netcdf is off
-  pkg_check_modules(ASAGI REQUIRED asagi) # asagi_nompi?
+  pkg_check_modules(ASAGI REQUIRED IMPORTED_TARGET asagi) # asagi_nompi?
   target_compile_definitions(SeisSol-lib PUBLIC USE_ASAGI)
   target_link_libraries(SeisSol-lib PUBLIC ${ASAGI_STATIC_LDFLAGS})
   target_include_directories(SeisSol-lib PUBLIC ${ASAGI_INCLUDE_DIRS})
```
Comment (sebwolf-de) [inline CMakeLists.txt:236]: Is there still code, which uses ASAGI directly in SeisSol? I think everything should be behind easi now?

Comment (Thomas-Ulrich) [reply CMakeLists.txt:236]: I think so too.

Comment (krenzland) [reply CMakeLists.txt:236]: AsagiModule.h for MPI init. Could be at some point moved to easi I guess

**Pull request merged**